brain_node:
  ros__parameters:
    game:
      team_id: 29 # consistant with GameControl
      player_id: 0 # 0 | 1 | 2 | 3
      field_type: "adult_size" # adult_size | kid_size | gdc
      player_role: "striker" # striker | goal_keeper
      player_start_pos: "left" # right | left
      
    robot:
      robot_height: 1.0
      odom_factor: 1.0
      vx_factor: 0.90
      yaw_offset: 0.0
      joystick: "logitech" # logitech | beitong
    
    enable_com: true

    rerunLog:
      enable: true
      server_addr: "rerun+http://127.0.0.1:9876/proxy"
      img_interval: 1
